src/javascript/ContentEditor/DesignSystem/OrderableValue/OrderableValue.jsx

```diff
@@ -1,24 +1,47 @@
 import {useTranslation} from 'react-i18next';
 import {useDrag, useDrop} from 'react-dnd';
+import clsx from 'clsx';
 import styles from '~/ContentEditor/utils/dragAndDrop.scss';
 import {Button, Close, HandleDrag} from '@jahia/moonstone';
-import React from 'react';
+import React, {useMemo} from 'react';
 import PropTypes from 'prop-types';
 
-export const OrderableValue = ({field, onFieldRemove, onValueReorder, index, component, isReferenceCard}) => {
-    const isDraggable = component ? component.props.isDraggable : false;
+const isDraggable = component => {
+    if (!component) {
+        return false;
+    }
+
+    if (typeof component.props.readOnly === 'boolean') {
+        return !component.props.readOnly;
+    }
+
+    if (typeof component.props.isDraggable === 'boolean') {
+        return component.props.isDraggable;
+    }
+
+    // Check if the component has multiple values
+    return (component.props.values?.[component.props.field?.name]?.length ?? 0) > 1;
```

Comment:
should we evaluate this first, i.e. prevent if component has one value but marked as isDraggable?